### PR TITLE
Telemetry Poller default VM metrics

### PR DESCRIPTION
**Pull request by jan-ciesla**
This PR adds metrics that are gathered by default by Telemetry Poller. Following metrics are updated:

* `vm_memory_total` - Total amount of currently allocated memory.
* `vm_memory_processes` - Amount of memory currently allocated for processes.
* `vm_memory_processes_used` - Amount of memory currently used for processes.
* `vm_memory_binary` - Amount of memory currently allocated for binaries.
* `vm_memory_ets` - Amount of memory currently allocated for ETS tables.
* `vm_total_run_queue_lengths_total` - A sum of all current run queue lengths.
* `vm_total_run_queue_lengths_cpu` - A sum of current CPU schedulers' run queue lengths.
* `vm_system_counts_process_count` - Number of process currently existing at the local node.

Following metrics are moved from `Count` to `Gauge` section:
* `sparrow_pools_warden_pools_gauge` - The current value of the selected measurement from the most recent event.
* `sparrow_pools_warden_workers_gauge{pool=${POOL}}` - Current number of workers operated by a given worker `POOL`.

Metrics collection was tested manually together with checking if they are correctly displayed correctly in grafana.

### CODE REVIEW

File: lib/mongoose_push/metrics/telemetry_metrics.ex [hunk 1]
```diff
@@ -27,24 +28,24 @@ defmodule MongoosePush.Metrics.TelemetryMetrics do
       ),
 
       # measurement is ignored in Counter metric
```
Comment (zofpolkowska) [inline lib/mongoose_push/metrics/telemetry_metrics.ex]:
```suggestion
      # Measurement is ignored in Counter metric
```

File: lib/mongoose_push/metrics/telemetry_metrics.ex [hunk 2]
```diff
@@ -27,24 +28,24 @@ defmodule MongoosePush.Metrics.TelemetryMetrics do
...
         tags: [:error_reason],
         tag_values: fn metadata -> %{metadata | error_reason: metadata.reason} end,
         description: "Counts the number of APNS state terminations"
       ),
-      Telemetry.Metrics.counter("mongoose_push.apns.state.get_default_topic.count",
+      counter("mongoose_push.apns.state.get_default_topic.count",
         description: "Counts the number of APNS default topic reads from the ETS cache"
       ),
 
       # sparrow events
```
Comment (zofpolkowska) [inline lib/mongoose_push/metrics/telemetry_metrics.ex]:
```suggestion
      # Sparrow events
```

**Review by zofpolkowska — COMMENTED** [review]
Just a tiny suggestion - let's unify the comments if we have to add them:) Otherwise - looks good to me.

**Review by kmakiela — APPROVED** [review]
Looks good, nice work catching this many spelling errors throughout the whole documentation

**Review by michalwski — APPROVED** [review]
Thanks for adding the Erlang VM metrics and fixing documentation.

**Comment by michalwski** [comment]
Nitpicking: the fixes to documentation could be done in a separete PR :)

**Pull request merged**